### PR TITLE
Fix admin interface bug errors

### PR DESCRIPTION
**Pull request by lucacug**

### CODE REVIEW

File: app/models/nomenclature_change/output.rb [hunk 1]
```diff
@@ -34,6 +34,7 @@
 # change.
 class NomenclatureChange::Output < ActiveRecord::Base
   include PgArrayParser
+  include ParsePgArray
```
Comment (WKocur) [inline app/models/nomenclature_change/output.rb]: Can this come out now?

Comment (lucacug) [reply app/models/nomenclature_change/output.rb]: I kept both cause I'm not totally sure the old one `PgArrayParser` has not still being used elsewhere.

Comment (WKocur) [reply app/models/nomenclature_change/output.rb]: Ah ok - I also noticed that the PgArrayParser is used in other files. Will this need changing there too? I haven't investigated properly.

**Comment by lucacug** [comment]
@WKocur I have actually branched this from yours and merged the `fix-checklist-test` one in

**Comment by WKocur** [comment]
@lucacug Are you happy for my changes to be merged in? If so then this should be fine.

**Comment by WKocur** [comment]
(I also haven't checked the test. Perhaps @Levia could take a quick peek at that?)

**Comment by Levia** [comment]
@WKocur What tests are you referring to specifically here?
Apart from that, is this PR ready to be merged in after I review it?

**Comment by WKocur** [comment]
@Levia I don't actually know... But whichever this branch fixes `fix-checklist-test` I'm not sure if that means more to you than me? From my perspective everything here looks fine!

**Comment by Levia** [comment]
@lucacug I've removed PgArrayParser completely.
Also, I had to add a migration to fix the `column_names` attribute in the `Trade::ValidationRule` model which it is supposed to be an array. This problem was mainly found on the test database, but also previous migrations weren't defining this attribute an array for some reason; so not sure what it actually made this an array in the first place.

**Comment by Levia** [comment]
Let's fix the remaining failing test in another branch

**Pull request merged**